--- cdcat-1.2-orig/src/cdcat.pro	2011-02-10 12:08:23.000000000 +0100
+++ cdcat-1.2/src/cdcat.pro	2011-03-08 15:52:24.994464509 +0100
@@ -93,7 +93,8 @@
 
 
 unix:distfiles.files +=   ../README_CSV_IMPORT ../Authors ../README ../ChangeLog \
-                          ../COPYING ../TRANSLATORS_README ../cdcat.png 
+                          ../README_IMPORT ../TRANSLATORS_README ../README_IMPORT.HU ../README_IMPORT.DE \
+			  ../cdcat.png 
 unix:distfiles.path =     /usr/local/share/cdcat
 unix:target.path +=       /usr/local/bin
 unix:translations.path += /usr/local/share/cdcat/translations
